graphqlapi/src/main/java/io/stargate/graphql/persistence/schemafirst/SchemaSourceDao.java

```diff
@@ -39,21 +40,23 @@
 // TODO purge old entries
 public class SchemaSourceDao {
 
-  public static final String TABLE_NAME = "graphql_schema";
-  @VisibleForTesting static final String KEY_COLUMN_NAME = "key";
+  public static final String KEYSPACE_NAME = "stargate_graphql";
+  public static final String TABLE_NAME = "schema_source";
+  @VisibleForTesting static final String NAMESPACE_COLUMN_NAME = "namespace";
   @VisibleForTesting static final String VERSION_COLUMN_NAME = "version";
   @VisibleForTesting static final String LATEST_VERSION_COLUMN_NAME = "latest_version";
   @VisibleForTesting static final String CONTENTS_COLUMN_NAME = "contents";
   @VisibleForTesting static final String APPLIED_COLUMN_NAME = "[applied]";
   private static final String DEPLOYMENT_IN_PROGRESS_COLUMN_NAME = "deployment_in_progress";
 
-  private static final Table EXPECTED_TABLE =
+  @VisibleForTesting
+  static final Table EXPECTED_TABLE =
       ImmutableTable.builder()
-          .keyspace("ks") // mock keyspace name, it won't be used
+          .keyspace(KEYSPACE_NAME)
```

Comment:
It still won't be used, but since it's known beforehand now, it doesn't hurt to use the right one.